pages/about.tsx

```diff
@@ -168,42 +202,73 @@ const AboutPage = () => {
           by the calling contract with{' '}
           <RelativeLink to="#3D" title="RETURNDATASIZE" /> and{' '}
           <RelativeLink to="#3E" title="RETURNDATACOPY" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H2>Gas costs</H2>} anchorKey="gascosts">
-        <p className="pb-6">
-          As an incentive to provide resources to run transactions, a fee is
-          paid to send and execute a transaction. The fee is determined by
-          several factors, including the amount of data sent or the amount of
-          work that a transaction requires. That fee is calculated through two
-          mechanisms. The first is a fixed cost that is defined by Ethereum,
-          depending on what is executed. The unit for that is called gas, and
-          remains the same for all transactions, though it can be changed by a
-          hardfork. The second component is the price of one gas unit, which
-          varies over time according to what people are willing to pay to run
-          their transactions. Its unit is ETH per gas.
+      <SectionWrapper header={<H2>Gas Costs</H2>} anchorKey="gascosts">
+        <p className="pb-4">
+          Each transaction on the Ethereum blockchain is vetted by a third-party
+          validator, before it is added to the blockchain. These validators are
+          compensated for conducting this vetting process, and adding
+          transactions to the blockchain, with incentive fee payments. Fees vary
+          from transaction to transaction, contingent on different variables for
+          different forks. Some variables in calculating fees include:
+          <ul className="list-disc mb-2">
+            <br></br>
+            <li>
+              <b>Current price of one gas unit:</b> Gas, or gwei, is a
+              denomination of Ethereum, used to in fee payment. Gas prices vary
+              over time, based on current demand for block space, measured in
+              ETH per gas.
+            </li>
+            <br></br>
+            <li>
+              <b>Calldata size:</b> Each calldata byte costs gas, the larger the
+              size of the transaction data, the higher the gas fees.
+            </li>
+            <br></br>
+            <li>
+              {' '}
+              <b>Intrinsic Gas</b>: Each transaction has an intrinsic cost of
+              21000 gas. Creating a contract costs 32000 gas, on top of the
+              transaction cost. Calldata costs 4 gas per byte equal to 0, and 16
+              gas for the others (64 before the hardfork <b>Istanbul</b>). This
+              cost is paid from the transaction before any opcode or transfer
+              execution.
+            </li>
+            <br></br>
+            <li>
+              <b>Opcode Fixed Execution Cost</b> : The fixed cost of executing
+              opcode if the transaction is sent to a contract, measured in gas.
```

Comment:
```suggestion
              opcode, measured in gas.
```
I don't think that helpful, and it confuses.

I would write it differently though, this is not clear at all:
"Each opcode has a fixed cost to be paid when it is executed. That cost is always the same for all executions, but can be changed in a new hardfork. See our reference."

Reply:
done, and added link